marimbabot_planning/src/utils.cpp

```diff
@@ -242,4 +249,33 @@ std::vector<CartesianHitSequenceElement> hit_sequence_absolute_to_relative(
     return hit_sequence_relative;
 }
 
+
+std::vector<CartesianHitSequenceElement> apply_chords(std::vector<CartesianHitSequenceElement> hits_relative)
```

Comment:
Could you add documentation here?

Reply:
And also for the other function that are introduced/have no documentation`? :)

Reply:
There is a doc string in the header file, that is shown if the IDE is configured properly, but I can copy it as we did with the other ones.

Reply:
If it is already there then nvm